MicrosoftTeams.schema.json

```diff
@@ -447,6 +447,20 @@
                 ]
             }
         },
+        "devicePermissions": {
+            "type": "array",
+            "description": "Specifies the devices the app requests from users.",
+            "maxItems": 5,
+            "items": {
+                "enum": [
+                    "geolocation",
+                    "media",
+                    "notifications",
+                    "midi",
+                    "openExternal"
```

Comment:
Lets stick to use capitalized values. Also the value was MidiSysex and not midi

Reply:
We don't use capitalized values anywhere else in the manifest, we use either all lowercase or camelCase.

I recommended midi over midiSysex because the Sysex part is just useless geekery.

Reply:
This is turning into another task module feature. Debo has already made corresponding changes in our backend settings store. @billbliss, @premmathi, can you help me understand why this list wasn't finalized weeks ago? We really need to get better at front loading these kinds of decisions so that devs don't have to redo completed work!

Reply:
I recommend we stick to the Spec. The permissions defined in camelCase. 

Reg midi vs midiSysex - these two are pretty much used interchangeably between Apple (midiSysex), Electron (midiSysex), W3C (midi), Windows (midi) and Android (midi) platforms. If using midiSysex saves us from doing another mapping I'm fine with us using it. HWA doesn't seem to support midi, so we will be also be saved from doing any other mappings regarding this in the future.

Reply:
Chatted with Yuri and Debo. We decided to use camelCasing and "midi".